pattern_library/monkey_utils.py

```diff
@@ -76,8 +76,13 @@ def node_render(context):
                     # Render result instead of the tag
                     return result
                 else:
+                    if default:
```

Comment:
Style nitpick: this doesn't need another level of identation, and could be
```python
elif default:
    return default
else:
    ...
```

Also, see my other comment about this not allowing falsey/null default values.